database/generateProducts.js

```diff
@@ -0,0 +1,64 @@
+const fs = require('fs');
+const csvWriter = require('csv-write-stream');
+const faker = require('faker');
+// const { sellers, writeThreeMillionSellers} = require('./generateSellers.js');
+
+const products = csvWriter({
+  headers: [
+    "id",
+    "name",
+    "condition",
+    "shippingFee",
+    "priceOriginal",
+    "priceActual",
+    "isOpenToOffers",
+    "category",
+    "style",
+    "brand",
+    "sellerID"
+
+  ]
+});
+
+const sellers = csvWriter({
+  headers: [
+    "id",
+    "name",
+    "address",
+    "isQuickShipper",
+    "joinYear",
+    "reviews",
+  ]
+});
+
+
+const writeTenMillionProducts = () => {
+  const start = new Date();
+  const conditions = ['Mint', 'Near Mint', 'Damaged'];
+  const trueOrFalse = [true, false];
+  const guitarCategories = ['Acoustic', 'Bass', 'Electric'];
+  const priceOriginal = (Math.floor(Math.random() * 21) + 20) * 100;
```

Comment:
You can also use `faker` to make random numbers, which I kind of like so I don't have to always type `Math.random(...`